### ISSUE TITLE
Feature request: Optional Add to Home for PSP/PSX/PSN Games

### ISSUE THREAD

**Issue by aknight2015** [blastrock/pkgj#416]
I apologize for using the "report issue" feature for this. I'd like the adding of PSX/PSP/PSN games to the home screen to be optional. I'm probably in the minority for this one though, but I'd thought I'd at least ask.
Since I have Adrenaline installed I don't really want to clutter up my home screen with non PSV games. I can't uninstall the bubble without uninstalling the game itself either.

**Comment by LiEnby** [comment]
If you don't have NoPspEmuDrm installed it will use the old adrenaline download instead, but maybe there should be a configuration option for this too. .

**Comment by aknight2015** [comment]
Yeah. Imagine your home screen after installing a few vita games and a few PSP games, a few PSX games, and a few PSM games. It's gonna get cluttered. But I understand what you're saying.

**Comment by symmetricalboy** [comment]
@aknight2015

I agree it should be optional, even though I prefer it to create bubbles.

In lieu of another solution, I offer you a slightly convoluted one that may please you for now:

Use https://anthe.studio/bubblestudio/ to edit your app.db file.
With this web app, you can delete a bubble. It does not uninstall the software, it just removes the database entry for the bubble from your LiveArea. You can clean it up manually this way. A database restore will always return everything that you delete.
You could also manually move everything to page 00000010 or above, if you don't want to be as destructive, but that is even more tedious & has proven to piss off the system sometimes & force a db refresh you didn't want.

Or maybe someone could modify the LiveArea to allow infinite pages & folders within folders? 😅

**Comment by aknight2015** [comment]
It's a temporary solution. I hope the developer of this app will implement the feature.

**Comment by LiEnby** [comment]
> @aknight2015
> 
> I agree it should be optional, even though I prefer it to create bubbles.
> 
> In lieu of another solution, I offer you a slightly convoluted one that may please you for now:
> 
> Use https://anthe.studio/bubblestudio/ to edit your app.db file. With this web app, you can delete a bubble. It does not uninstall the software, it just removes the database entry for the bubble from your LiveArea. You can clean it up manually this way. A database restore will always return everything that you delete. You could also manually move everything to page 00000010 or above, if you don't want to be as destructive, but that is even more tedious & has proven to piss off the system sometimes & force a db refresh you didn't want.

Faster way: rename ux0:pspemu/PSP/GAME to ux0:/pspemu/PSP/GAME2 

Delete the game from the livearea. 

Rename GAME2 back to GAME .

**Comment by symmetricalboy** [comment]
> Faster way: rename ux0:pspemu/PSP to ux0:/pspemu/PSPA
> 
> Delete the game from the livearea.
> 
> Rename PSPA back to PSP .

That's f'in' brilliant.
I ***do*** tend to take the long way around...

> Annoyingly page id's above 9 are used for folders. So increasing the number of pages would also break all the folders and or duplicate entries with them.

Huh.
That... suddenly illuminates why it was proving inconsistent & tumultuous to use that addressing space. 😅

**Comment by HackZy01** [comment]
> [Annoyingly page id's above 9 are used for folders. So increasing the number of pages would also break all the folders and or duplicate entries with them.

I mean it would still be nice if it meant removing folder support for people who prefer more pages
